packages/suite/src/config/wallet/networks.ts

```diff
@@ -426,6 +431,9 @@ type Network = {
     features?: string[];
     label?: ExtendedMessageDescriptor['id'];
     tooltip?: ExtendedMessageDescriptor['id'];
+    support?: {
+        [key: number]: string;
+    };
```

Comment:
minor - nitpicks - fell free to resolve on your own

My personal preference would be more explicit name, something like `minFirmwareVersion`.

The type could be more specific
```suggestion
    support?: {
        [key: 1 | 2]: VersionInput;
    };
```

Reply:
My first option of choice was making it the very same way like [coins.json](https://github.com/trezor/connect/blob/cfeb056f1d3c6b2f40fd45bb20e63931dea3f57b/src/data/coins.json#L46)

```
  "support": {
                "connect": true,
                "suite": true,
                "trezor1": "1.5.2",
                "trezor2": "2.0.5"
            },
```

So thats for naming  it`support`, my idea was to make it is consistent with coins.json. It is true that I am using different keys - this is to make it easy to use with typescript. Also setting literal keys as you suggest would be nice, but `major_version` field is typed as number and If I am not mistaken, it wouldn't be assignable to number type.

Reply:
I see, my idea of `minFirmwareVersion` name is from [releases.json](https://github.com/trezor/trezor-suite/blob/develop/packages/connect-common/files/firmware/1/releases.json) file.

As I said, I'm fine with the `support` naming too...